src/app/globals.css

```
@@ -2,120 +2,150 @@
+    --radius-lg: var(--radius);
+    --radius-xl: 0.75rem;
+    --radius-full: 9999px;
+
+    /* Shadows (sparingly — borders do the structural work) */
+    --shadow-sm: 0 1px 2px 0 rgb(0 0 0 / 0.05);
+    --shadow: 0 1px 3px 0 rgb(0 0 0 / 0.1), 0 1px 2px -1px rgb(0 0 0 / 0.1);
+    --shadow-md: 0 4px 6px -1px rgb(0 0 0 / 0.1), 0 2px 4px -2px rgb(0 0 0 / 0.1);
+    --shadow-lg: 0 10px 15px -3px rgb(0 0 0 / 0.1), 0 4px 6px -4px rgb(0 0 0 / 0.1);
+
+    /* Type foundation (Inter is loaded via next/font in layout.tsx) */
+    --font-sans: "Inter", ui-sans-serif, system-ui, -apple-system, "Segoe UI", Roboto, sans-serif;
```

Stylelint (17.7.0)

[error] Expected "Roboto" to be "roboto" (value-keyword-case)

(value-keyword-case)